janus_core/md.py

```diff
@@ -360,20 +360,23 @@ def _optimize_structure(self) -> None:
                 self.logger.info("Minimizing at step %s", self.dyn.nsteps)
             optimize(self.struct, **self.minimize_kwargs)
 
-    def configure_filenames(self) -> None:
+    def configure_filenames(self, pressure: Optional[float] = None) -> None:
```

Comment:
Is there a way to do this without explicitly passing pressure? Given that pressure can be an attribute, it might be nice if it could be used directly rather than passing it around, but maybe this would end up messier than it needs to.

Reply:
i think we could if configure_filenames is overridden in NPT, which would duplicate most of the code. 

Or if in configure_filenames we check the type and if it is NPT we add the pressure.

We could also have a function, say ```self.parameters()``` which would format the parameters (T, P) for the file names in which case the NPT would override to add the pressure instead of passing it in and the default in MolecularDynamics add just the temperature.  That would save duplicating the rest of the code.

Will give it a think/go later

Reply:
I have a feeling this created new problems for a similar issue when setting up the headers, but adding pressure with default as None in the main MD class might be an option?

Reply:
yes that would be another way around